.github/workflows/ci-build.yaml

```diff
@@ -0,0 +1,98 @@
+name: Integration tests
+on: 
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+
+  build-docker:
+    name: Ensure Docker image builds
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build Docker image
+        run: |
+          DOCKER_PUSH=false make image
+
+  build-go:
+    name: Build & cache Go code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Golang
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.14.2'
```

Comment:
we might want to match the go version in argocd ci which is 1.14.12 currently.

Reply:
Done.